utils/create-token.js

```diff
@@ -6,7 +6,10 @@ const generateToken = async (userId, email, params) => {
     : params.EXPIRES_IN
       ? { expiresIn: params.EXPIRES_IN }
       : {};
-  const token = await jwt.sign({ userId, email }, process.env.SECRET, options);
+  const secret =    params && params.useConfirmationSecret
```

Comment:
I think it is not the best idea to configure this function via params, just  rewrite the `generateToken` to get a secret as an argument

Reply:
I cannot make fundemental changes, cause this may affect the work of others.